docs/USING.rst

```diff
@@ -50,3 +50,41 @@ to hard-disk booting the installed system.
 This is, as the filepath states, merely an example of how you might deploy to
 physical nodes.  Existing deployment systems such as Canonical's MAAS have also
 been run succesfully.
+
+Using CURL Commands
+====================
+
+Included herwith are some examples about interacting with HaaS API using the curl 
+utility.
+
+** Example for registering a Node which use IPMI **
+Node name: dummyNoderHaaS-02
+Ipmi info: 
+hostname:-           ipmiHost4node-02
+ipmi_username:-      ipmiUser4node-02
+ipmi_password:-      ipmiPass4node-02
+
+
+
+For nodes using IPMI use the following api call:
+
+curl -X PUT http://127.0.0.1:5001/node/dummyNoderHaaS-02 -d '
+> {"obm": { "type": "http://schema.massopencloud.org/haas/v0/obm/ipmi",
+> "ipmi_host": "ipmiHost4node-02",
```

Comment:
@henn also raise this point, so I guess there are more people who think that variable names should be simple. I agree and have made the necessary changes. Committed the changes.